### ISSUE TITLE
Wrong results after Tensorflow 2x model conversion

### ISSUE THREAD

**Issue by sergiocasaspastor** [shepnerd/inpainting_gmcnn#70]
Hi,

I hope you still give support for this repository. I tried converting this repository model to Tensorflow 2x step by step. I think everything was fine, but when training and testing the model the results were not very satisfactory. So, maybe there is a problem with my conversion that I can't find it. I tried to train the model on a small subset of OpenImages V6. The principal parameters that I used (sorry, because names can be slightly different than yours):

--img_size 256x256x3
--batch_size 4
--learning_rate 1e-4
--gaussian_steps 7
--gaussian_kernel_size 32
--gaussian_kernel_std 20.0
--reconstruction_loss_weight 1.2
--adversarial_loss_weight 0.001
--gradient_penalty_loss_weight 10
--id_mrf_loss_weight' 0.03
--nn_stretch_sigma 0.5
--id_mrf_style_weight 1.0
--id_mrf_content_weight 1.0

I pretrained the model with only confidence reconstruction loss for the recommended steps, and results for this phase seem fine.

![imagen](https://user-images.githubusercontent.com/39574343/111080389-8aca3780-84fe-11eb-8726-35875be2fc20.png)
![imagen](https://user-images.githubusercontent.com/39574343/111080394-8ef65500-84fe-11eb-9689-7641eeb1b8bb.png)
![imagen](https://user-images.githubusercontent.com/39574343/111080399-9289dc00-84fe-11eb-8967-8ce61940f97f.png)

However, in the training phase, after some steps, the results do not seem to converge and eliminate the mask.
![imagen](https://user-images.githubusercontent.com/39574343/111080418-adf4e700-84fe-11eb-8c9c-821f1f21d99c.png)
![imagen](https://user-images.githubusercontent.com/39574343/111080424-b3523180-84fe-11eb-8a23-86b391a47052.png)
![imagen](https://user-images.githubusercontent.com/39574343/111080428-b816e580-84fe-11eb-8602-0f25e6cee8cd.png)

I would like to know if you have any idea of what could be happening, or you have experienced any similar issue. I reviewed the code, networks, and losses many times, but could not find any solution.

Thank you very much.
